core/src/filed/backup.cc

```diff
@@ -176,9 +167,10 @@ bool BlastDataToStorageDaemon(JobControlRecord* jcr,
 
   AccurateFinish(jcr); /* send deleted or base file list to SD */
 
-  StopHeartbeatMonitor(jcr);
+  //  StopHeartbeatMonitor(jcr);
+
+  if (sd != nullptr) { sd->signal(BNET_EOD); /* end of sending data */ }
 
-  sd->signal(BNET_EOD); /* end of sending data */
```

Comment:
Try to not alter this function as far as possible, so also this should stay as it was